webapp/channels/src/actions/websocket_actions.jsx

```diff
@@ -217,7 +219,7 @@ function restart() {
 
 export function reconnect() {
     // eslint-disable-next-line
-    console.log('Reconnecting WebSocket');
+    console.log(`Reconnecting WebSocket - ${getCurrentTime()}`); //eslint-disable-line no-console
```

Comment:
I'm totally fine leaving this, but Chrome and Firefox (maybe Safari too, but I never checked it) both have options to show timestamps for every message in the JS console so we might not actually need this

Reply:
you are right @hmhealey , it even show the miliseconds, was not aware of it, will remove this then 👍 
<img width="391" alt="Screenshot 2024-12-18 at 10 30 36 AM" src="https://github.com/user-attachments/assets/97861597-2a36-442b-85cc-64fad1f0e3fe" />